Patch to address issue seen with BIND.

Will pass to upstream in due course.

diff -r 01304a701517 src/unix/udp.c
--- a/src/unix/udp.c	Fri Mar 24 08:28:22 2023 -0700
+++ b/src/unix/udp.c	Fri Mar 24 09:25:03 2023 -0700
@@ -495,7 +495,8 @@
     if (setsockopt(fd, SOL_SOCKET, SO_REUSEPORT, &yes, sizeof(yes)))
        return UV__ERR(errno);
   }
-#elif defined(SO_REUSEPORT) && !defined(__linux__) && !defined(__GNU__)
+#elif defined(SO_REUSEPORT) && !defined(__linux__) && !defined(__GNU__) && \
+	!defined(__sun__)
   if (setsockopt(fd, SOL_SOCKET, SO_REUSEPORT, &yes, sizeof(yes)))
     return UV__ERR(errno);
 #else
